src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/Public.Data/DataSetVersionMappingControllerTests.cs

```diff
@@ -370,6 +371,242 @@ originalCountryMappingToUpdate with
             updatedMappings.LocationMappingPlan.Levels.AssertDeepEqualTo(
                 expectedFullMappings,
                 ignoreCollectionOrders: true);
+
+            // Assert that the batch saves still show the location mappings as incomplete, as there
+            // are still mappings with type "None" and "AutoNone" in the plan.
```

Comment:
should this comment read ` ... are still mappings with type "AutoNone" in the plan`. I think `None` won't be present will it?

Reply:
Updated ta, plus updated the initial test setup to be more realistic